dotnet/samples/GettingStartedWithVectorStores/Step1_Ingest_Data.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.VectorData;
+using Microsoft.SemanticKernel.Connectors.InMemory;
+using Microsoft.SemanticKernel.Embeddings;
+
+namespace GettingStartedWithVectorStores;
+
+/// <summary>
+/// Example showing how to generate embeddings and ingest data into an in-memory vector store.
+/// </summary>
+public class Step1_Ingest_Data(ITestOutputHelper output, VectorStoresFixture fixture) : BaseTest(output), IClassFixture<VectorStoresFixture>
+{
+    /// <summary>
+    /// Example showing how to ingest data into an in-memory vector store.
+    /// </summary>
+    [Fact]
+    public async Task IngestDataIntoInMemoryVectorStoreAsync()
+    {
+        // Construct the vector store and get the collection.
+        var vectorStore = new InMemoryVectorStore();
+        var collection = vectorStore.GetCollection<string, Glossary>("skglossary");
+
+        // Ingest data into the collection.
+        await IngestDataIntoVectorStoreAsync(collection, fixture.TextEmbeddingGenerationService);
+
+        // Retrieve an item from the collection and write it to the console.
+        var skDefinition = await collection.GetAsync("4");
```

Comment:
Name seems odd to me, why `skDefinition`? Would `record` or `item` be better?

Reply:
I used skDefinition because that's what's in the record that I'm retrieving.  Record 4 is the definition of Semantic Kernel.
I can see how this is not obvious, so renaming to record.